mw-webapp/src/logic/wayPage/labels/LabelLine.module.scss

```diff
@@ -2,21 +2,40 @@
 
 .labelLine {
   align-items: baseline;
-  gap: $gapBig;
+  gap: 0 $gapBig;
```

Comment:
why 0

Reply:
We don't require vertical paddings on mobile versions.

![bug_03_01](https://github.com/user-attachments/assets/41edd883-1d4b-4d6c-9147-986ccffbb8e5)

![bug_03_02](https://github.com/user-attachments/assets/80273326-8bef-4af6-af02-beadb9f37fae)